rerun_py/rerun_sdk/rerun/notebook.py

```diff
@@ -7,6 +7,7 @@
 from typing import TYPE_CHECKING, Any, Literal
 
 import numpy as np
+import pkg_resources
```

Comment:
I think `pkg_resource` is part of `setuptools` on which we don't explicitly depend. Any chance you could do the same with `importlib`?

Reply:
Seems like `importlib.util.find_spec` does the same thing